tests/integration/targets/nxos_static_routes/tests/common/rtt.yml

```diff
@@ -51,25 +49,22 @@
                       - forward_router_address: 192.0.2.55
                         tag: 1
                         admin_distance: 1
-        state: overridden
+        state: replaced
```

Comment:
Round trip tests should ideally happen with `state: overridden`, any reason why we're switching to replaced here?

Reply:
Same as above, overridden is risky!

Reply:
Added my reply in previous comment.